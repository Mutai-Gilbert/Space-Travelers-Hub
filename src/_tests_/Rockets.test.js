import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store';
import Rockets from '../components/Rockets/Rockets';

describe('Test Rocket components', () => {
  it('renders correctly', () => {
    const tree = renderer
      .create(
        <Provider store={store}>
          <Rockets />
        </Provider>,
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
