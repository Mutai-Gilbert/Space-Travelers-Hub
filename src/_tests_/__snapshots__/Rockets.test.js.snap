// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Rocket components renders correctly 1`] = `null`;
